sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -242,7 +275,7 @@ def get_media_settings(key, media_dict):
             return media_settings
```

Comment:
Unrelated to your change but can you please change this error message to notice?
```
line 315:
        helper_logger.log_error("Error: No values for physical port '{}'".format(physical_port))

Reply:
updated